services/web/server/src/simcore_service_webserver/diagnostics_settings.py

```diff
@@ -3,37 +3,35 @@
 from typing import Dict
 
 from aiohttp.web import Application
-from pydantic import BaseSettings, Field, NonNegativeFloat, PositiveFloat, validator
+from pydantic import Field, NonNegativeFloat, PositiveFloat, validator
 from servicelib.aiohttp.application_keys import APP_CONFIG_KEY
+from settings_library.base import BaseCustomSettings
 
 
-class DiagnosticsSettings(BaseSettings):
+class DiagnosticsSettings(BaseCustomSettings):
 
-    slow_duration_secs: PositiveFloat = Field(
+    DIAGNOSTICS_SLOW_DURATION_SECS: PositiveFloat = Field(
         1.0,
         description=(
             "Any task blocked more than slow_duration_secs is logged as WARNING"
             "Aims to identify possible blocking calls"
         ),
-        env="AIODEBUG_SLOW_DURATION_SECS",
+        env=["DIAGNOSTICS_SLOW_DURATION_SECS", "AIODEBUG_SLOW_DURATION_SECS"],
```

Comment:
question: in the future, will we be able to get rid of these double env vars? have you got a plan for it?

Reply:
Fields with multiple ``env`` defined are necessary to keep backwards compatibility. 
NOTE: when using ``env``, we must always put then the name of the field first (should add a check for that)